docs/posts/execute_commands_on_host.md

```diff
@@ -91,6 +93,28 @@ if [ -n "${ZSH_VERSION-}" ]; then
 fi
 ```
 
+### fish
+
+Place this in a new fish config (such as `~/.config/fish/conf.d/cnf_handle.fish`:
```

Comment:
[The official `fish` documentation](https://fishshell.com/docs/current/language.html#autoloading-functions) recommends that functions go in `~/.config/fish/functions` and it should be the name of the function:

`~/.config/fish/functions/fish_command_not_found.fish`

I might reword the sentence as:

> Place this snippet in a new fish configuration file (`~/.config/fish/functions/fish_command_not_found.fish`)

Reply:
sure, go ahead.

i was deciding whether I should change to `~/.config/fish/functions`, but because my own testing was done with `~/.config/fish/conf.d` I waited until I get access to my Linux laptop again so I can test.